scripts/west_commands/runners/nrf_common.py

```diff
@@ -254,9 +256,12 @@ def program_hex(self):
         # What type of erase/core arguments should we pass to the tool?
         core = None
 
-        if self.family == 'NRF54H_FAMILY':
+        if self.family in ('NRF54H_FAMILY', 'NRF92_FAMILY'):
             erase_arg = 'ERASE_NONE'
 
+            cpuapp = self.build_conf.getboolean('CONFIG_SOC_NRF54H20_CPUAPP') or self.build_conf.getboolean('CONFIG_SOC_NRF9280_CPUAPP')
+            cpurad = self.build_conf.getboolean('CONFIG_SOC_NRF54H20_CPURAD') or self.build_conf.getboolean('CONFIG_SOC_NRF9280_CPURAD')
```

Comment:
We could use more generic conditions here. How about checking the board qualifiers for `/cpuapp` etc?

Reply:
Fair point. We can try to improve this in future PRs.